From cdb0ac145e497bd40245e257fb5e974bbb96895e Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Wed, 14 Mar 2018 15:25:46 +0800
Subject: [PATCH 3484/5242] MLK-17798: ARM64: dts: configure
 IOMUXD_COMP_CTL_GPIO_1V8_3V3_GPIORHB

commit  5f24f5dc4ba9af6a3e449edb3d12b2764325dd5e from
https://source.codeaurora.org/external/imx/linux-imx.git

The ESAI and SPDIF pin are in enet bank, the board is using 3.3v,
so we need to configure the PSW_OVR to zero, whose default setting
is for 2.5V.

Signed-off-by: Anson Huang <anson.huang@nxp.com>
Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |    1 +
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts  |    1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index 93c2bc5..7aa7f1b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -154,6 +154,7 @@
 		pinctrl_hog: hoggrp {
 			fsl,pins = <
 				SC_P_ENET0_REFCLK_125M_25M_LSIO_GPIO5_IO09 0xc600004c
+				SC_P_COMP_CTL_GPIO_1V8_3V3_GPIORHB_PAD	0x000514a0
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 8181103..55ef477 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -201,6 +201,7 @@
 		pinctrl_hog: hoggrp {
 			fsl,pins = <
 				SC_P_MCLK_OUT0_ADMA_ACM_MCLK_OUT0	0x0600004c
+				SC_P_COMP_CTL_GPIO_1V8_3V3_GPIORHB_PAD	0x000514a0
 			>;
 		};
 
-- 
1.7.9.5

